### PR TITLE
Update stackdriver webhook to allow customize alarm

### PR DESCRIPTION
**Pull request by bcanyelles**
Update stackdriver webhook to allow customize alarm from documentation field.

According the  [google cloud docs](https://cloud.google.com/monitoring/alerts/doc-variables), the `documentation` field can be customized using variables. So you can send a json in this field and process it to overwite all fields you want.

### CODE REVIEW

**Comment by satterly** [comment]
Cool. This looks really useful. Can you add a test that uses an example "documentation" in the [webhooks test case for Stackdriver](https://github.com/alerta/alerta/blob/master/tests/test_webhooks.py#L640)? Thanks.

File: tests/test_webhooks.py [hunk 1]
```diff
@@ -640,15 +649,16 @@ def test_riemann_webhook(self):
     def test_stackdriver_webhook(self):
 
         # open alert
+        import nose.tools; nose.tools.set_trace()
```
Comment (satterly) [inline tests/test_webhooks.py]: You need to remove this line and the line below and the tests will pass. Thanks.

File: tests/test_webhooks.py [hunk 2]
```diff
@@ -640,15 +649,16 @@ def test_riemann_webhook(self):
...
         response = self.client.post('/webhooks/stackdriver', data=self.stackdriver_open, headers=self.headers)
+        import nose.tools; nose.tools.set_trace()
```
Comment (satterly) [inline tests/test_webhooks.py]: To be removed.

**Comment by satterly** [comment]
@bcanyelles Don't worry about the failed tests. I'll merge this and fix them. Thanks!

**Comment by bcanyelles** [comment]
I'm sorry @satterly , I forget it

**Pull request merged**